test-plan/eventing/control-plane/event-delivery/00-prepare.yaml

```diff
@@ -0,0 +1,56 @@
+# Namespace for executing conformance tests.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: conformance
+
+---
+
+# Role for the recordevents Pod.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: conformance
+  name: conformance-recordevents
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list"]
+
+---
+
+# RoleBinding for the recordevents Pod.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: conformance-recordevents
+  namespace: conformance
+subjects:
+- kind: ServiceAccount
+  namespace: conformance
+  name: default
+roleRef:
+  kind: Role
+  name: conformance-recordevents
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+# InMemoryChannel for the Channel/Subscription tests.
+#
+# Should be replaced with other Channel implementations
+# that want to be tested for conformance.
+apiVersion: messaging.knative.dev/v1
+kind: InMemoryChannel
```

Comment:
One of the goals of the conformance tool is to be able to run it against any backing channel implementation. It should be possible to meet this goal by just creating a `Channel` object. By default the backing channel implementation in the InMemoryChannel so nothing should change other than replacing `InMemoryChannel` by `Channel` everywhere in this doc.